File: kotlin-ide/j2k/new/testData/newJ2k/constructors/privateConstructors.java
class C {
  private C(int arg1, int arg2, int arg3) {
  }

  private C(int arg1, int arg2) {
    this(arg1, arg2, 0);
  }

  public C(int arg1) {
    this(arg1, 0, 0);
  }
}

File: j2k/tests/testData/fileOrElement/constructors/privateConstructors.kt
// ERROR: Too many arguments for internal fun C(arg1: kotlin.Int): C defined in root package
// ERROR: Too many arguments for internal fun C(arg1: kotlin.Int): C defined in root package
fun C(arg1: Int): C {
    return C(arg1, 0, 0)
}

class C private(arg1: Int, arg2: Int, arg3: Int = 0)
